proto/voice-info.proto

```diff
@@ -0,0 +1,175 @@
+syntax = "proto3";
+
+package org.sim.voices;
+
+/*
+  Version info, shall adhere to semantic versioning in Hex number for easier
+  comparison.  E.g. 0x00010001 for version 1.0.1.
+  Rules are as follows:
+    - Major number shall be increased, in case of incompatible changes,
+        e.g. semantically incompatible change, new parameters needed, etc.
+    - Minor number shall be increased, in case of a compatible changes,
+        e.g. renaming of members, adding new members, changing the protobuf
+        type in a compatible way, improving a models speed/quality, etc.
+    - Patch number shall be increased, in case of a backwards-compatible
+      bug-fix
+*/
+message Revision {
+  uint32 version = 1;
+}
+
+/* Describes a voice model's file entry */
+message FileEntry {
+  enum Compression {
+    COMPRESSION_PLAIN = 0;      // no compression
+    COMPRESSION_BROTLI = 1;
+    COMPRESSION_BZIP2 = 2;
+    COMPRESSION_GZIP = 3;
+    COMPRESSION_ZIP = 4;
+  }
+  enum Format {
+    FORMAT_PLAIN = 0;           // plain file, no archive
+    FORMAT_TAR = 1;             // tar archive
+    FORMAT_CPIO = 2;            // cpio archive
+  }
+  string path = 1;                // relative path, mandatory
+  Compression compression = 2;    // file compression, optional
+  Format format = 3;              // file format, optional
+  string md5 = 4;                 // MD5 sum of file, mandatory
+}
+
+/* Speech model language / locale info */
+message Language {
+  /* <ISO 639 language code>_<ISO 3166 country code> */
+  enum Locale {
+    /* default: any language/country) */
+    LOCALE_ANY = 0;
+    is_IS = 1;
+    /* more entries can be appended here ... */
+  }
+  Locale locale = 1;  // locale of the voice, mandatory
+  string name = 2;    // language name of the voice, mandatory
+}
+
+/* Phonemes used for acoustic model, alphabets and symbol mappings */
+message PhonemeDescription {
+  enum PhonemeType {
+    PHONEME_TYPE_INVALID = 0;
+    /* use IPA symbols, these are always distinct single symbols */
+    PHONEME_TYPE_IPA = 1;
+    /* use "extended Sampas", these are mostly made up of multiple symbols */
+    PHONEME_TYPE_XSAMPA = 2;
+  }
+  /* Mapping between Alphabet and model representation */
+  message AlphabetMapping {
+    /*
+      Mapping of all IPA/XSampa symbols to their corresponding model value,
+      as well as meta-symbols like <sp> "short pause", <pau> "pause", etc. used
+      in model training.
+    */
+    map<string, uint32> symbols = 1;
+  }
+  PhonemeType phoneme_type = 1; // phoneme type, mandatory
+  AlphabetMapping mapping = 2;  // symbol mappings, mandatory
+  bool syllabified = 3;         // "." at each syllable boundary, optional
+  bool stress_labels = 4;       // "0" or "1" after each vowel, optional
+  string word_separator = 5;    // word separator; default "-", optional
```

Comment:
What is this for?

Reply:
e.g. to define if the voice was trained using stress and syllabification symbols

Reply:
My comment was supposed to reference only line 76, i.e. the `word_separator`.  It's just a special symbol for a word bounary in a phoneme sequence?

Reply:
yes, I'm not sure though, if that param exactly is ever used in training - I implemented it in the g2p for inspection purposes, i.e. if the user wants to extract transcriptions of single tokens from running text, for example.

Reply:
@bnika : could you elaborate on that ? I was just addiing the entries used in the new normalizer